common/lib/xmodule/xmodule/course_module.py

```diff
@@ -335,6 +335,15 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_(
+            "Enter the URL for the official course Facebook group. "
+            "If you provide a URL, the edX mobile app includes a button that students can tap to access the group."
+        ),
```

Comment:
Well theoretically, since both our server platform and our apps are open source now, yes, a developer can enable the social features on their instance of the server and their fork of the apps, set the correct configurations, and their app's social features would work.